app/main.py

```diff
@@ -2,0 +1,17 @@
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    mv, source_file, destination_file = split_command
+    path = destination_file[0:destination_file.rfind("/")]
+    if mv == "mv":
+        if "/" in command:
+            if not os.path.exists(os.path.join(path)):
+                os.makedirs(os.path.join(path))
```

Comment:
Code Efficiency Issue: The use of `os.path.join(path)` is redundant here. You can just use `os.path.exists(path)`. Same applies for `os.makedirs(path)`.